README.md

```diff
@@ -1,2 +1,62 @@
+- `issue-comment`
+
+**Action inputs**:
+
+| Name                                   | Description                                                                | Default                                                                                                                                                    |
+|----------------------------------------|----------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tos-location`                         | Terms Of Service location (URL or relative file)                           |                                                                                                                                                            |
+| `signature-branch`                     | *Optional*: Git branch where the signature file will be stored             | `main`                                                                                                                                                     |
+| `signature-location`                   | *Optional*: Signature file location                                        | `.github/gradle-enterprise-tos.json`                                                                                                                       |
+| `pr-comment-tos-approval-missing`      | *Optional*: PR comment added when Terms of Service are not approved        | `Please accept [Gradle Enterprise Terms Of Service](<tos-location>) to get your PR Build Scan published by commenting this PR with the following message:` |
+| `pr-comment-tos-approval-request`      | *Optional*: PR comment to approve the Terms of Service                     | `I have read Gradle Enterprise Terms Of Service and I hereby accept the Terms`                                                                             |
+| `pr-comment-tos-approval-confirmation` | *Optional*: PR comment added when Terms of Service are approved            | `All Contributors have accepted Gradle Enterprise Terms Of Service.`                                                                                       |
+| `white-list`                           | *Optional*: CSV List of users not required to approve the Terms of Service |                                                                                                                                                            |
+
+
+**Usage**:
+
+```yaml
+name: Gradle - Terms of Service approval verification
+
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+
+permissions:
+  # required to update signature file
+  contents: write
+  # required to comment PR
+  pull-requests: write
+  # required to update PR status check
+  actions: write
+  statuses: write
+
+jobs:
+  gradle-check-tos:
+    uses: gradle/github-actions/.github/workflows/check-tos.yml@v1.0
```

Comment:
This shouldn't be `v1.0` for the test. Instead, just link to `check-tos.yml@main`.

Reply:
I don't get the point, this is the official usage snippet.
[The test is using the version from the same ref](https://github.com/gradle/github-actions/blob/880f23f61553e3e67cd07d2df3f2bac44f3ddc92/.github/workflows/test-check-tos.yml#L13)

Reply:
What I mean is that we should not even have a `v1.0` tag in the repository at this stage, since in GitHub Actions land this indicates a 1.0 release.

I recommend instead creating a "floating" tag that we force push with every release. Normally I'd suggest a `v1` tag that points at the latest `1.x` release, but I'm not sure what to do for pre-release.

Reply:
But you're right: this isn't a test reference, it's in the documentation. Perhaps just use `v1` in the docs, and create a `v1` tag that points at the most recent release (0.3?).

Reply:
Yes, let's keep `v1.0` in the docs and we'll adjust the tags when 1st iteration will be complete to make this `v1.0` a real thing.
The current tags (up to `v0.4`) are just here for testing purpose and keep being overwritten.